src/build.d

```diff
@@ -813,15 +810,7 @@ auto sourceFiles()
                 .filter!(e => !e.canFind("stub", "optabgen.c"))
                 .chain(targetCH.only)
                 .array,
-        tkC:
-            dirEntries(env["C"], "*.{c,h}", SpanMode.shallow)
-            .map!(e => e.name)
-            .array,
-        backendObjects:
-            dirEntries(env["C"], "*.c", SpanMode.shallow)
-                .map!(e => e.baseName.stripExtension)
-                .filter!(e => !e.canFind("stub", "optabgen", "cgcv", "cgobj", "newman"))
-                .chain(targetObjs)
+        backendObjects: ["fp", "strtold", "tk"]
```

Comment:
The automatic file listing was only added to decrease the overhead of this script to the conversion process. Now that there are only three files remaining, we can simplify this (and hopefully soon remove it).